pkg/hintrunner/zero/zerohint_uint256.go

```diff
@@ -190,3 +192,64 @@ func createSplit64Hinter(resolver hintReferenceResolver) (hinter.Hinter, error)
 
 	return newSplit64Hint(a, low, high), nil
 }
+
+func newUint256SqrtHint(n hinter.ResOperander, root hinter.ResOperander) hinter.Hinter {
+	// from starkware.python.math_utils import isqrt\nn = (ids.n.high << 128) + ids.n.low\nroot = isqrt(n)\nassert 0 <= root < 2 ** 128\nids.root.low = root\nids.root.high = 0
+	return &GenericZeroHinter{
+		Name: "Uint256Sqrt",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> from starkware.python.math_utils import isqrt
+			//n = (ids.n.high << 128) + ids.n.low
+			//root = isqrt(n)
+			//assert 0 <= root < 2 ** 128
+			//ids.root.low = root
+			//ids.root.high = 0
+
+			nLow, nHigh, err := GetUint256AsFelts(vm, n)
+			if err != nil {
+				return err
+			}
+			//> n = (ids.n.high << 128) + ids.n.low
+			valueLowU256 := uint256.Int(nLow.Bits())
+			value := uint256.Int(nHigh.Bits())
+			value.Lsh(&value, 128)
+			value.Add(&value, &valueLowU256)
+
+			//> root = isqrt(n)
+			calculatedUint256Root := uint256.Int{}
+			calculatedUint256Root.Sqrt(&value)
+			calculatedUint256RootBytes := calculatedUint256Root.Bytes()
+			calculatedFeltRoot := fp.Element{}
+			calculatedFeltRoot.SetBytes(calculatedUint256RootBytes)
+			//> assert 0 <= root < 2 ** 128
+			if !utils.FeltIsPositive(&calculatedFeltRoot) {
+				return fmt.Errorf("assertion failed: a = %v is out of range", calculatedUint256Root)
+			}
+			rootAddr, err := root.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+			//> ids.root.low = root
+			rootLowValue := memory.MemoryValueFromFieldElement(&calculatedFeltRoot)
+			err = vm.Memory.WriteToAddress(&rootAddr, &rootLowValue)
+			if err != nil {
+				return err
+			}
+			//> ids.root.high = 0
+			rootHighValue := memory.MemoryValueFromFieldElement(new(fp.Element).SetInt64(0))
```

Comment:
Could you use FeltZero constant from utils here?

Reply:
done